std/datetime.d

```diff
@@ -29929,7 +29929,7 @@ else version(Windows)
         }
 
 
-        this(string name, TIME_ZONE_INFORMATION tzInfo) @safe immutable pure
+        this(string name, TIME_ZONE_INFORMATION tzInfo) @trusted immutable pure
```

Comment:
I believe that the consensus at this point is that if the @safety depends on the argument being correct, then it should be marked as `@trusted`. The reality of the matter in this case is that either will work. The constructor is private, and the function calling it is already `@trusted`. So, just mark it as `@trusted`.

And if the Windows API gives a `wchar*` that _isn't_ zero-terminated, then it's just plain broken, since it's a C API. But actually, in this case, it doesn't even matter, because the `TIME_ZONE_INFORMATION` is constructed by std.datetime in the caller, and it's explicitly zero-terminated.